### PR TITLE
Fixes in ClusterServiceVersion manifest

### PR DESCRIPTION
**Pull request by swghosh**
Fix CSV manifests:
- trailing newline in `operators.openshift.io/valid-subscription` annotation which is required yaml lint
- description in base had erroneous `'displayName:..'` which was removed
- populated config/samples with examples of each resource type
    - during bundle phase it adds the `alm-examples` annotation to CSV
    - which helps avoid `WARN[0000] ClusterServiceVersion validation: [OperationFailed] provided API should have an example annotation`

Signed-off-by: Swarup Ghosh <swghosh@redhat.com>

### CODE REVIEW

File: config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: '[]'
```
Comment (thejasn) [inline config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml:5]: You can still keep this. It will get over written.

Comment (swghosh) [reply config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml]: Okay, done 👍🏻

**Comment by thejasn** [comment]
/label docs-approved
/label px-approved
Only minor bundle changes, no impact on docs/px.

**Comment by swghosh** [comment]
/cc @xingxingxia 
for qe-approved label

**Pull request merged**